test/unit/setup/after-env/index.ts

```diff
@@ -3,3 +3,9 @@ import "@testing-library/jest-dom"
 import "./mock-network"
 import "./polyfill-dom"
 import "./polyfill-fetch"
+import {configure} from "@testing-library/react"
+import env from "app/core/env"
+
+if (env.isCI) {
+  configure({asyncUtilTimeout: 5000})
```

Comment:
just curious if this significantly increases the time it takes for tests to run when they are failing?

Reply:
Good question. This is only for our component tests. If there are many of them and they all are failing then it would increase the time by 5x. If that happens we can have CI "bail early" if a single test fails (or if 5 or 10 fail).